owncloudComLibrary/src/main/java/com/owncloud/android/lib/common/OwnCloudClient.java

```diff
@@ -163,7 +161,7 @@ private void setRequestId(HttpBaseMethod method) {
         // Header to allow tracing requests in apache and ownCloud logs
         addHeaderForAllRequests(OC_X_REQUEST_ID, requestId);
 
-        Log_OC.d(TAG, "Executing " + method.getClass().getSimpleName() + " in request with id " + requestId);
+        Timber.d("Executing " + method.getClass().getSimpleName() + " in request with id " + requestId);
```

Comment:
I'm not sure if this makes sense `method.getClass().getSimpleName()`
Timber loge automatic the class name